api/TransformStream.json

```
@@ -87,6 +87,56 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "cancel": {
+          "__compat": {
+            "description": "`transformer.cancel` method",
```

Given that `cancel` is the key in the parameter object passed to the `TransformStream` constructor, it confuses me slightly that this is called a method (even if the spec uses this terminology). How about:

```suggestion
        "transformer_cancel_callback": {
          "__compat": {
            "description": "`transformer.cancel` callback",
```

---

The MDN articles for the [ReadableStream](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream/ReadableStream#underlyingsource)/[WritableStream](https://developer.mozilla.org/en-US/docs/Web/API/WritableStream/WritableStream#underlyingsink)/[TransformStream constructors](https://developer.mozilla.org/en-US/docs/Web/API/TransformStream/TransformStream#transformer) as well as [the streams guides](https://developer.mozilla.org/en-US/docs/Web/API/Streams_API/Using_readable_streams#the_readablestream_constructor) also talk about "methods", so I tried to be consistent with those. Although the specification does use the name [`TransformerCancelCallback`](https://streams.spec.whatwg.org/#callbackdef-transformercancelcallback) for the *type* of that method... 😅

I think I'll stick with "method" for now. I'll make the feature name a bit more specific though, like you suggested.